src/caustics/lenses/multiplane.py

```diff
@@ -177,8 +177,10 @@ def raytrace(
         ----------
         x: Tensor
             angular x-coordinates in the image plane.
+        *Unit: radians*
```

Comment:
The spacing doesn't match the rest of the usages.

Reply:
Updated